src/components/appointments/CreateAppointment/CreateAppointment.spec.js

```diff
@@ -110,9 +113,7 @@ describe('Create appointment session expiration', () => {
 
     // now finally let the session timeout in 1s
     act(() => updateSessionExpiry(1));
-    await waitFor(async () => {
-      await screen.findByText('Your session has expired');
-    });
+    await screen.findByText('Your session has expired', undefined, {timeout: 2000});
```

Comment:
failed because of the internal timeout being too low, so the session-will-expire-over-1000ms-and-then-render-a-different-page cycle exceeded that. Giving 1s for the rendering etc. is sufficient here.